### ISSUE TITLE
SCC New Concerns - Phone Call Recap

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#638]
In GitLab by @kdjstudios on Jan 10, 2018, 16:51

I just got off the phone with Garrett with SCC, and he is voicing more concerns now and almost all of them are questions for functionality that to my knowledge has never been voiced or is a part of SAB. He seems very unpleased with the system and how things are going. I worked with him to the best of my ability and informed him we would be discussing his concerns tomorrow morning with the support team. I did also mention that it would be beneficial if we all could jump on a call tomorrow to further discuss and insure everyone is on the same page. I have listed below the concerns he voiced today:

 

1.       Time line? When can he go live? He was not happy with how long it’s taking.

a.       I was sure to mention that a lot of the delay was due to him not responding to our inquiries, which would halt progress.

2.       Mass rate increases? How does he perform a mass rate increase across multiple accounts.

a.       Every year they go through a majority of accounts and perform a rate increase across all accounts at once. It is that time of year. He wants to know if he should do that in his current system or if SAB can perform this. I informed him that I would research this and follow up, but off hand I was not aware of this type of functionality.

3.       Reports. He claims none of the reports ever worked for him until today while we were on the phone.

a.       In addition to this, he needed to have his password reset because he was trying to use a wrong password. Might be part of the problem?

b.      He asked a few questions about comparing this year’s data to last year’s data, and this month to last year same month, etc. I informed him that we are currently working on new reports and adding additional functionality to existing reports, but that this current is not available.

4.       Quarterly accounts: He is open to transferring the accounts to a standard calendar year billing. But has questions on how the transition would work. Proration, billing, etc.

a.       If we do convert them to SAB compatible accounts, would this bump up the go live timeline?

5.       Second site:

a.       This would be all new accounts and he would like to know more about the pricing and how that would work with his existing package.

i.      I informed him it would integrate directly into the same user interface and really is just a matter of finding the cost. I let him know I would be reaching out to Dave to schedule a call to discuss that further.

6.       Mainly he is concerns that his current billing system provides more functionality and features then SA Billing does.

 

I will be sure to bring this up in our conference tomorrow for internal discussion, but when is the best time to jump on a call with Garrett to discuss the follow up with him tomorrow?

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 11, 2018, 13:19

After discussion today, I called Garrett and left a message to call me back with an operator there. I also sent over in two emails the following to posts to recap everything that was discussed.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 11, 2018, 13:19

> Hello Garrett,
> 
> The support team has been able to determine that we should be ready to have you begin your process into using SA Billing in a parallel environment. In order to insure that the SA Billing system is in sync with your existing billing system we will need the files listed below. It is imperative that these are exports performed right after your billing cycle completion, but before you make any changes, payments, credits, etc. This also needs to happen after you have performed your rate increases as discussed. This will insure that SA Billing matches your system. Once the SA Billing import is completed then you may start performing your tasks billing tasks in both systems. This will allow you the opportunity to learn a majority of the functionality of SA Billing. On the next billing cycle you can then run both systems in parallel and then choose to send your invoices our through SA Billing. 
> 

> Import Files requested:
> •	invoice_detail.xlsx and pdf format (Comparison file/their invoice total)

> •	cmr1.xls (List of Customer and their address and phone numbers)

> •	tas2.xls (Package and handle link)

> •	TUSACC12_.txt (Usage File)

> •	rcr1.xls (fixed/recurring fee list)

> •	tas1.csv (usage charges)

> •	Tax Report

> •	Billing_frequency.xlsx

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 11, 2018, 13:20

Hello Garrett,

After a discussion with our support team this morning, we do have some updates. I wanted to first send you these update in email form as I have had a few conference calls pop up on me today. I will be giving you a call later today to further discuss with you too. 

Mass Rate increase:

1.	This is not a function of SA Billing and we have not had any previous request to this functionality.
2.	If you would like this functionality we can discuss the custom development and costs to programing this functionality. This would be developed after you have gone live in SA Billing. 

Reports:

1.	You inquired about comparison reports, such as year to year, month to month, etc.
2.	These type of reports are not part of SA Billing currently, however we are open to suggestions for improving the reports. For these we will need detailed explanation of the reports you would like to see. These reports would be added to our wish list of new features and functionality and addressed accordingly.

Second Site:

1.	You inquired about adding an additional site and you described that these will be new accounts for a new site. 
2.	We will need to discuss on a conversation with Dave Kravitz about the cost to add an additional site.

Nonstandard Quarterly, Semiannual, and Annual accounts:

1.	You described that you had Quarterly, Semiannual, and Annual accounts, however you do not bill in a standard calendar year, which would make these custom billing periods, not quarterly, Semiannual, and Annual accounts. 
2.	You have the option to convert these to standard Quarterly, Semiannual, and Annual accounts or to setup additional sub billing cycles for each account accordingly. 

Go Live:

1.	I have sent over an additional email describing the process we will start to go live and the documents that we require from you in order to progress. We will want to start the Go Live process immediately.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 11, 2018, 13:31

Posting updates to another ticket for the Go Live specifically: #631

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 11, 2018, 13:42

I will be trying to schedule a call with Dave, Garrett, and I to further discuss the Second site on Monday. 

We will need to move forward with converting his accounts to standard quarterly, etc accounts and help him find how do insure prorations and such are handled appropriately. 

We have the data needed to start the Go Live.

Rate increase we will be further discussing upon how successful the Go Live is. 

Reports we will discuss after we Go Live.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Aug 23, 2018, 10:21

Updated Priority to 3 for SCC

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 13, 2018, 14:30

Topics are irrelevant now.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Nov 13, 2018, 14:34

closed
